recipes/tmtk/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "tmtk" %}
+{% set version = "0.4.4.dev3" %}
+{% set sha256 = "d94600bd60b120ae96ef9e6fad5019114836bf6da0bc181113ab7433a2d04f75" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py<34]
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
```

Comment:
It looks like there are fairly specific restrictions on the versions of a number of the dependencies. These should be reflected in the recipe as well, unless you specifically know that the latest version of all listed packages is ok:

https://github.com/thehyve/tmtk/blob/master/requirements.txt

Reply:
The latest should be fine indeed. Tests frequently run against latest versions.